homeassistant/components/upnp/strings.json

```diff
@@ -1,25 +1,22 @@
 {
   "config": {
+    "flow_title": "UPnP/IGD: {name}",
     "step": {
-      "confirm": {
-        "description": "Do you want to set up UPnP/IGD?"
+      "init": {
+      },
+      "ssdp_confirm": {
+        "description": "Do you want to set up this UPnP/IGD device?"
```

Comment:
One thing that I was thinking last night is that most people don't know what UPnP/IGD is, or how it comes that Home Assistant discovers one. I think that we might want to extend this description to explain what it is/what it can be used for. What do you think ?

Reply:
Agreed. Perhaps something like:
> A router has been discovered in your network. You can monitor the traffic statistics for this device. Do you want to set up this integration?

Reply:
Be aware though: Some other devices also provide information via UPnP/IGD profile, such as access points.